com.example.WebPConverter.json

```diff
@@ -0,0 +1,62 @@
+{
+  "app-id": "com.example.WebPConverter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "webp_converter.py",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--filesystem=xdg-config/gtk-3.0:ro",
+    "--filesystem=xdg-config/gtk-4.0:ro",
+    "--talk-name=org.freedesktop.portal.Desktop",
+    "--own-name=com.example.WebPConverter"
```

Comment:
1. the dbus name for the app-id is granted by default
2. the app-id is wrong anyway so neither are acceptable.